src/makeInfinityAdder.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
-}
+  return function add(x) {
+    if (x || x === 0) {
```

Comment:
```suggestion
    if (x !== undefined) {
```
Won't it work the same?